docs/background/topics/layout.rst

```diff
@@ -26,9 +26,7 @@ When a refresh is requested on a container, any sub-containers will also be refr
 Length units
 ============
 
-Toga uses CSS units in its public API. All size calculations assume that the user has a
-96 DPI display; if the user has a higher density display, the sizes that are applied
-will be scaled accordingly. In practice, this means that:
+Toga uses CSS units in its public API. Their physical size depends on the device type:
 
```

Comment:
I think privileging 96 DPI in this context causes more confusion than it solves. If the user wants a full explanation, they can always follow the link.